saleor/graphql/decorators.py

```diff
@@ -72,3 +90,21 @@ def check_perms(context):
 staff_member_required = account_passes_test(
     lambda context: context.user.is_active and context.user.is_staff
 )
+
+
+def check_attribute_required_permissions():
```

Comment:
Please add docstrings to these decorators. They will be reused in more places in the future so it's good that they're documented.